### DISCUSSION TITLE
Electron + Nextjs + Tailwind no refresh in dev

### DISCUSSION THREAD

**Discussion by jo-wp** [saltyshiomix/nextron#161] [General]
Hello,

I would like to know, why my npm run dev, 
No refresh my edits.

**Comment by jo-wp** [comment]


**Reply by m5x5** [reply]
I believe you can delete discussions too here 🤔

**Comment by saltyshiomix** [comment]
@jo-wp 

Tailwindcss examples are updated!
Please see:
- https://github.com/saltyshiomix/nextron/issues/243
- https://github.com/saltyshiomix/nextron/issues/209
